src/components/TeacherComponents/OnboardingForm/OnboardingForm.tsx

```diff
@@ -212,15 +194,15 @@ const OnboardingForm = ({
             $flexDirection="column"
           >
             <OakPrimaryButton
-              disabled={!props.canSubmit}
+              disabled={!props.canSubmit || isSubmitting}
```

Comment:
I originally had this entering a loading state when the form was submitted. But that was weird when the "Skip" button was clicked. It made it look like the "Continue" button had been clicked and we had ignored the preference.

I've opted to disable both while the form is submitting as it looked even weirder if both buttons had a loading spinner.